client/webserver/site/src/js/markets.ts

```diff
@@ -884,8 +885,11 @@ export default class MarketsPage extends BasePage {
       this.setMaxOrder(mkt.maxSell.swap)
       return
     }
+    if (mkt.maxSellRequested) return
```

Comment:
This is the critical bit that really resolves https://github.com/decred/dcrdex/issues/1612 for me